src/Polytopes/Cone/properties.jl

```diff
@@ -114,12 +102,9 @@ julia> ambient_dim(C)
 ambient_dim(C::Cone) = pm_cone(C).CONE_AMBIENT_DIM
 
 """
-    codim(C)
-
-Returns the codimension of a cone.
+    codim(C::Cone)
 
-# Arguments
-- `C::Cone`: A cone.
+Returns the codimension of `C`.
```

Comment:
```suggestion
Return the codimension of `C`.
```
(Of course this was "wrong" before, so not your "fault", but since you corrected this elsewhere in this PR...)